### ISSUE TITLE
The nodata.png is not centered on the screen - fix

### ISSUE THREAD

**Issue by judismith** [dotnet-presentations/dotnet-maui-workshop#128]
In the video at 3:43:45 the image on nodata.png is displayed as being centered on the screen. The code demonstrated does not result in a centered image. Additionally, the code in the Read Me also does not result in the image being centered. However, the following code does work:

`<CollectionView.EmptyView>
            <Grid HorizontalOptions="Fill" VerticalOptions="Fill">
                  <Image Source="nodata.png"
                              HeightRequest="160"
                              WidthRequest="160"
                              Aspect="AspectFill">
                  </Image>
              </Grid>
</CollectionView.EmptyView>`

**Comment by jamesmontemagno** [comment]
Yeah, I think i made adjustments to it as I was able to get it centered on some operating systems and not on others.
